python/README.md

```diff
@@ -1,64 +1,22 @@
 
-To use the `aida_interchange` package within your project you must first install it using the instructions described in the [Install setup.py](install-setup.py) section. It is recommended that you install `aida_interchange` into a virtual environment for development purposes. To install `aida_interchange`  within your virtual environment first activate your virtual environment and run the install command below. See [Python Development Setup](python-development-setup) for more details on creating a virtual environment.
+To use the `aida_interchange` package within your project, you must first install it. It is recommended that you install `aida_interchange` into a python3 virtual environment. See [Python Virtual Environment README](VIRTUAL-README.md) for more details on creating and using a virtual environment.
 
-# Install setup.py
+## Install setup.py
 
-To install `aida_interchange`, run the following command within the `python` directory:
+To install `aida_interchange`, make sure a [Python Virtual Environment](VIRTUAL-README.md) is activated, navigate to the `AIDA-Interchange-Format/python` directory, and run the following command:
 
 ```bash
-python3 setup.py install
+$ python3 setup.py install
 ```
 The `aida_interchange` modules can now be imported into your project.
 
-# Python Development Setup
+## API Documentation
 
-Setting up a virtual environment for Python development is the recommended best way to mitigate problems between multiple systems working on this project.
-
-## Install virtualenv
-
-```bash
-$ cd ~
-$ mkdir .virtualenvs
-$ sudo pip install virtualenv
-```
-Verify virtualenv is installed
-
-```bash
-$ which virtualenv
-```
-
-## Create virtual environment
-
-Navigate inside the `python` directory and run the following
-
-```bash
-$ virtualenv -p python3 ~/.virtualenvs/aida-interchange-format
-$ source ~/.virtualenvs/aida-interchange-format/bin/activate
-$ pip install -r requirements.txt
-```	
-
-## Activate virtual environment
-
-Your virtual environment will be automatically activated after running the virtualenv command. If you need to re-activate your virtual environment you can run the following command while in the root python folder of the project
-
-```bash
-$ source ~/.virtualenvs/aida-interchange-format/bin/activate
-```
-
-## Deactivate virtual environment
-
-To deactivate your current virtual environment run the following command.
-
-```bash
-$ deactivate
-```
-
-# Documentation
-
-The python project uses [Sphinx](http://www.sphinx-doc.org/en/master/) for generating documentation. To generate the documentation, navigate to the `python/docs` directory and run the `update_documentation.sh` script.
+The python project uses [Sphinx](http://www.sphinx-doc.org/en/master/) for generating documentation. To generate the documentation, make sure a [Python Virtual Environment](VIRTUAL-README.md) is activated, navigate to the `AIDA-Interchange-Format/python/docs` directory, and run the `update_documentation.sh` script.
```

Comment:
The instructions say `navigate to the `AIDA-Interchange-Format/python/docs directory`, but the bash instructions below include `cd docs`. Should the instructions be updated to `navigate to the `AIDA-Interchange-Format/python directory`? 

This may be an issue in the verdi repo as well.

Reply:
I did this intentionally.  Since the README is at the level above the docs folder, the instructions are relative to the location of the README.  If you think this is unclear, I'm happy to take a suggestion to make it clearer.

Reply:
No that is okay, I just wanted to point it out.